Day-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,14 @@
-import React from 'react';
+import React, {Fragment} from 'react';
 import './Button.css';
+
+function ActionButton(props) {
+    return (
+        <Fragment>
+            <div className={`actionButton ${props.buttonColor} ${props.buttonWidth}`} onClick={() => props.whenClicked(props.value)}>
+                {props.value}
+            </div>
+        </Fragment>
```

Comment:
Why use a fragment here when everything is already wrapped in a div? Lose the fragment since it's unnecessary.